webapp/channels/src/components/widgets/icons/fa_next_icon.tsx

```diff
@@ -7,12 +7,14 @@ import {defineMessage} from 'react-intl';
 
 import LocalizedIcon from 'components/localized_icon';
 
+import {t} from 'utils/i18n';
+
 type Props = {
     additionalClassName?: string;
 }
 
 const iconTitle = defineMessage({
-    id: 'generic_icons.next',
+    id: t('generic_icons.next'),
```

Comment:
Is this needed? I think `mmjstool` can properly detect i18n strings using `defineMessage`

Reply:
I thought so too but the pipeline wouldn't pass without it: https://github.com/mattermost/mattermost/actions/runs/6627549647/job/18002705272

I think it might have broken from a recent merge or something. Not sure why it wasn't happening to anyone else though

Reply:
That's odd. I guess that's another reason for us to get away from our custom i18n tooling and `mmjstool` then towards the standard FormatJS stuff